get_aqi.py

```diff
@@ -119,6 +152,25 @@ def get_aqi_color(aqi_level, country):
         elif aqi_level == "very high":
             # Red : 101 < aqi
             return "255 0 0"
+    elif country == "US":
+        if aqi_level == "Good":
+            # Green : 0 < aqi <= 50
+            return "121 227 71"
+        elif aqi_level == "Moderate":
+            # Yellow : 51 < aqi <= 100
+            return "251 255 93"
+        elif aqi_level == "Unhealthy for Sensitive Groups":
+            # Orange : 101 < aqi <= 150
+            return "226 138 54"
+        elif aqi_level == "Unhealthy":
+            # Red : 151 < aqi <= 200
+            return "234 51 36"
+        elif aqi_level == "Very Unhealthy":
+            # Purple : 201 < aqi < 300
+            return "126 63 185"
+        elif aqi_level == "Hazardous":
+            # Maroon :  aqi < 301
```

Comment:
Should be the opposite
```suggestion
            # Maroon : 301 < aqi
```